members/S001194.yaml

```diff
@@ -2,21 +2,21 @@ bioguide: S001194
 contact_form:
   method: post
   action: /contact
+  usePhantomJsLogging: true
```

Comment:
why do we have to force phantom?

Reply:
I think we should as I noticed the dropdown menu uses JS. 

See the `onchange` jquery-esque value in the `select id="actions"` element in the image below. 

![image](https://user-images.githubusercontent.com/8680733/41688668-0d3bd5ea-74bb-11e8-9a73-2891b1ef6e30.png)

I could be wrong, but wanted to play it safe with this change hence why I added phantom. 

@tomdionesotes

Reply:
The form uses JS to fetch the rest of the inputs etc once we select a value in `select#actions`, sure. Lots of other forms work like this, and AFAIK their yamls work fine without forcing phantom.

Once we select a value in `select#actions` we can check if the rest of the elements are loaded with wait / find - unless the find is returning a false positive. 

You can test for a find false positive by temporarily editing this yaml to remove the `select#actions` click step, add a find for an input like first name, and see if the find succeeds in the tester.

Reply:
Oh ok got it. I did the test you suggested and it failed to find `firstname`, so at least now I know that other elements were not loading before a element in `select#actions` was selected. 

I also tested the changes w/o phantom and they passed. Taking phantom out, are you OK with the `find` and `click-on` changes in this yaml? 

@tomdionesotes